deploy/olm-catalog/portworx/1.0.3/core_v1alpha1_storagecluster_crd.yaml

```diff
@@ -33,35 +33,55 @@ spec:
               type: string
             imagePullPolicy:
               type: string
-            imagePullSecret: {}
-            secretsProvider: {}
-            startPort: {}
+            imagePullSecret:
+              type: string
+            secretsProvider:
+              type: string
+            startPort:
+              type: integer
+              format: int32
             updateStrategy: {}
-            revisionHistoryLimit: {}
+            revisionHistoryLimit:
+              type: integer
+              format: int32
             placement: {}
             kvdb:
               properties:
                 internal:
                   type: boolean
-                endpoints: {}
+                endpoints:
+                  type: array
+                  items:
+                    type: string
                 authSecret:
                   type: string
             storage:
               properties:
-                useAll: {}
-                useAllWithPartitions: {}
-                forceUseDisks: {}
+                useAll:
+                  type: boolean
+                useAllWithPartitions:
+                  type: boolean
+                forceUseDisks:
+                  type: boolean
                 devices: {}
-                journalDevice: {}
-                systemMetadataDevice: {}
-                dataStorageType: {}
-                raidLevel: {}
+                journalDevice:
+                  type: string
+                systemMetadataDevice:
+                  type: string
+                dataStorageType:
+                  type: string
+                raidLevel:
```

Comment:
Does `raidLevel` level values other than int? I see in the CRD it's defined as a string. Just checking if it needs to a string.

Reply:
Not that I know of but we store raid level as string. Will keep it as string for extensibility.